src/librustc/session/config.rs

```diff
@@ -1401,7 +1401,7 @@ options! {DebuggingOptions, DebuggingSetter, basic_debugging_options,
     thinlto: Option<bool> = (None, parse_opt_bool, [TRACKED],
         "enable ThinLTO when possible"),
     inline_in_all_cgus: Option<bool> = (None, parse_opt_bool, [TRACKED],
-        "control whether #[inline] functions are in all cgus"),
+        "control whether `#[inline]` functions are in all cgus"),
     tls_model: Option<String> = (None, parse_opt_string, [TRACKED],
         "choose the TLS model to use (rustc --print tls-models for details)"),
```

Comment:
quotes around the command line: `` choose the TLS model to use (`rustc --print tls-models` for details) ``

If you could try grepping for command lines elsewhere (e.g., `rustc --`, `cargo --`, `rls --` or even something more flexible), that would be great.